ivy/functional/frontends/paddle/tensor/creation.py

```diff
@@ -66,3 +66,10 @@ def arange(start, end=None, step=1, dtype=None, name=None):
 @to_ivy_arrays_and_back
 def empty(shape, dtype=None):
     return ivy.empty(shape=shape, dtype=dtype)
+
+
+@to_ivy_arrays_and_back
+def empty_like(x, dtype=None, name=None):
+    if dtype:
+        return ivy.empty(shape=x.shape, dtype=dtype)
+    return ivy.empty(shape=x.shape, dtype=x.dtype)

```

Comment:
Not sure why we're making a call to `ivy.empty` and not `ivy.empty_like`? Also I think we can refactor this to make it more readable but having just one call to `ivy.empty_like` and handling the `dtype` before that in a single statement using the ternary operator.

Reply:
Yes, you are correct, thanks